### PR TITLE
Improving Travis and Docker integration

### PR DESCRIPTION
**Pull request by alaxalves**

### CODE REVIEW

**Comment by alaxalves** [comment]
@jywarren @icarito @emilyashley could you guys review this?

File: scripts/install-deps.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "deb http://packages.laboratoriopublico.org/publiclab/ stretch main" > /etc/apt/sources.list.d/publiclab.list
+
+# Obtain key
+mkdir -p ~/.gnupg
+echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
```
Comment (icarito) [inline scripts/install-deps.sh]: I realize this comes from Dockerfile but I wonder if line 8 would be enough (ipv4.pool...).

File: scripts/install-deps.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                        python-gdal \
+                        libgdal-dev \
+                        g++ \
+                        curl \
+                        procps \
+                        git \
+                        imagemagick \
+                        zip
+
+sed -i 's/<policy domain="delegate" rights="none" pattern="HTTPS" \/>//g' /etc/ImageMagick-6/policy.xml
```
Comment (icarito) [inline scripts/install-deps.sh]: This is a security measure that should not be relevant in .travis.yml as it doesn't process outside requests.
The goal of this PR is to improve test times, yes?
If it does, looks good to me!

**Review by icarito — APPROVED** [review]
Looks fine to me, thanks!

File: scripts/install-deps.sh (hunk 1 above)
Comment (alaxalves) [reply scripts/install-deps.sh]: What me to drop it and test it out? @icarito

File: scripts/install-deps.sh (hunk 2 above)
Comment (alaxalves) [reply scripts/install-deps.sh]: Great, I didn't quite know what was going on here, thx for pointing that out. I'll make those adjustments

**Comment by alaxalves** [comment]
@icarito Just updated the PR and the build remained successful. Thanks a lot for the input

**Review by jywarren — APPROVED** [review]

**Pull request merged**